hexrd/ui/resources/ui/image_mode_widget.ui

```diff
@@ -60,71 +60,19 @@
        <string>Raw</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
-       <item row="0" column="1">
-        <layout class="QFormLayout" name="formLayout">
-         <item row="0" column="0" colspan="2">
-          <widget class="QCheckBox" name="raw_threshold_mask">
-           <property name="text">
-            <string>Set Threshold Mask</string>
-           </property>
-          </widget>
-         </item>
-         <item row="1" column="0">
-          <widget class="QComboBox" name="raw_threshold_comparison">
-           <property name="enabled">
-            <bool>false</bool>
-           </property>
-           <item>
-            <property name="text">
-             <string>less than</string>
-            </property>
-           </item>
-           <item>
-            <property name="text">
-             <string>greater than</string>
-            </property>
-           </item>
-           <item>
-            <property name="text">
-             <string>equal to</string>
-            </property>
-           </item>
-          </widget>
-         </item>
-         <item row="1" column="1">
-          <widget class="ScientificDoubleSpinBox" name="raw_threshold_value">
-           <property name="enabled">
-            <bool>false</bool>
-           </property>
-           <property name="keyboardTracking">
-            <bool>false</bool>
-           </property>
-           <property name="decimals">
-            <number>8</number>
-           </property>
-           <property name="minimum">
-            <double>-100000.000000000000000</double>
-           </property>
-           <property name="maximum">
-            <double>100000.000000000000000</double>
-           </property>
-          </widget>
-         </item>
-        </layout>
-       </item>
        <item row="0" column="0">
-        <layout class="QVBoxLayout" name="verticalLayout_3">
```

Comment:
I think it might be better if these checkboxes are stacked vertically instead of horizontally, with `Tabbed View` being on top.

Reply:
Sounds good, fixed.